samples/update-backup-schedule.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2024 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START spanner_update_backup_schedule]
+async function updateBackupSchedule(
```

Comment:
This file should be part of samples directory.

Reply:
It's under the correct directory. GitHub UI doesn't group directories together, so it may seem like this file is under the `system-test` directory.